develop-docs/sdk/expected-features/spotlight.mdx

```diff
@@ -0,0 +1,414 @@
+- `"on"`
+- `"1"`
+
+### Falsy Values
+
+The following values should be treated as `spotlight: false` (disabling Spotlight):
+
+- `"false"`
+- `"f"`
+- `"n"`
+- `"no"`
+- `"off"`
+- `"0"`
+
+### URL String Values
+
+Any other string value should be treated as a Spotlight backend URL, enabling Spotlight with that URL.
+
+**Example parsing logic (Python reference):**
+
+```python
+def parse_spotlight_env(env_value):
+    if not env_value:
+        return None
+    
+    env_value_lower = env_value.lower().strip()
+    
+    # Truthy values
+    if env_value_lower in ("true", "t", "y", "yes", "on", "1"):
+        return True
+    
+    # Falsy values
+    if env_value_lower in ("false", "f", "n", "no", "off", "0"):
+        return False
+    
+    # Any other value is treated as a URL
+    return env_value
+```
+
+## Precedence Rules
+
+The interaction between configuration options and environment variables follows these precedence rules:
+
+1. **Config option takes precedence over env var**, except:
+   - If `spotlight: true` (boolean, no URL specified) AND `SENTRY_SPOTLIGHT` is set to a URL string → use the env var URL
+   - This allows developers to enable Spotlight via config but override the URL via environment variable
+
+2. **If `spotlight` is set to a string URL** → it overrides the env var completely
+
+3. **If using two-attribute approach:**
+   - If `spotlightUrl` config and env var are both set → prefer config value (`spotlightUrl`)
+   - If `spotlight: false` is explicitly set → ignore `spotlightUrl` value and the env var
+
+4. **If `spotlight: false` explicitly set** → ignore env var and any URL configuration
+
+**Precedence Examples:**
+
+```python
+# Example 1: Config boolean true + env var URL → use env var URL
+# Config: spotlight: True
+# Env: SENTRY_SPOTLIGHT=http://custom:3000/stream
+# Result: Enabled with http://custom:3000/stream
+
+# Example 2: Config URL string → always use config URL
+# Config: spotlight: "http://config:3000/stream"
+# Env: SENTRY_SPOTLIGHT=http://env:3000/stream
+# Result: Enabled with http://config:3000/stream
+
+# Example 3: Config false → disable regardless of env var
+# Config: spotlight: False
+# Env: SENTRY_SPOTLIGHT=http://localhost:8969/stream
+# Result: Disabled
+
+# Example 4: Two-attribute approach with spotlightUrl
+# Config: spotlightUrl: "http://config:3000/stream"
+# Env: SENTRY_SPOTLIGHT=http://env:3000/stream
+# Result: Enabled with http://config:3000/stream (config takes precedence)
+```
+
+## Data Collection Behavior
+
+When Spotlight is enabled, SDKs must implement the following data collection behavior:
+
+### Envelope Transmission
+
+- Send a copy of **every envelope** to the Spotlight server
+- This includes errors, transactions, sessions, profiles, replays, and all other envelope types
+- The Spotlight server HTTP semantics are the same as the Sentry server HTTP semantics (e.g. use POST to send envelopes)
+
+### Sampling
+
+- Enable **100% sample rate** for the Spotlight pipeline
```

Comment:
I'm pretty sure some SDKs don't meet that requirement. For example, on Cocoa, we have two transports. One sends to Sentry, the other to Spotlight. Sampling happens before the transport. So, currently, the sampling behavior is the same for both Spotlight and Sentry envelopes. I'm unsure if having two different sampling behaviours is actually worth the complexity. This could be tricky to set up in some SDKs. It could also confuse users.